src/components/footer/Footer.js

```diff
@@ -1,12 +1,17 @@
-import './styles.css';
+import "./styles.css";
 import React from "react";
+import { FaFacebookSquare, FaPhoneAlt } from "react-icons/fa";
+import { BsInstagram } from "react-icons/bs";
 
 const Footer = () => {
-    return (
+return (
     <div className="footer">
-        <p>Template Footer</p>
+    
+    <div className="social-icons">
+    <a href="0471 65 34 02"><FaPhoneAlt style={{ color: "white", fontSize: "30px" }}  /></a> <a href="example.com"><FaFacebookSquare style={{ color: "white", fontSize: "30px" }} /></a> <a href="example.com"> <BsInstagram style={{ color: "white", fontSize: "30px" }} /></a>
```

Comment:
You could clean this up a little bit by using the `IconContext`: https://github.com/react-icons/react-icons#configuration